### PR TITLE
Add inspectionEnabled to tunnel contracts

### PR DESCRIPTION
**Pull request by jfullerton44**
### Changes proposed: 
- Adds inspectionEnabled field to all tunnel contracts

### Other Tasks:
- [x] If you updated the Go SDK did you update the PackageVersion in tunnels.go

### CODE REVIEW

File: cs/src/Contracts/Tunnel.cs [hunk 1]
```diff
@@ -127,6 +127,12 @@ public Tunnel()
     [MaxLength(TunnelMaxPorts)]
     public TunnelPort[]? Ports { get; set; }
 
+    /// <summary>
+    /// Gets or sets if inspection is enabled for the tunnel.
+    /// </summary>
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public bool? InspectionEnabled { get; set; }
```
Comment (IlyaBiryukov) [inline cs/src/Contracts/Tunnel.cs:134]: nit: `IsInspectionEnabled`

Comment (jfullerton44) [reply cs/src/Contracts/Tunnel.cs]: updated

File: cs/src/Contracts/Tunnel.cs [hunk 2]
```diff
@@ -127,6 +127,12 @@ public Tunnel()
     [MaxLength(TunnelMaxPorts)]
     public TunnelPort[]? Ports { get; set; }
 
+    /// <summary>
```
Comment (IlyaBiryukov) [inline cs/src/Contracts/Tunnel.cs:130]: Why set it on the tunnel object and not on `TunnelOptions`? Tunnel options also apply to tunnel ports, and so can be controlled with more granularity.

**Review by IlyaBiryukov — APPROVED** [review]

File: cs/src/Contracts/Tunnel.cs (hunk 2 above)
Comment (derekbekoe) [reply cs/src/Contracts/Tunnel.cs:130]: Should this be inside of `TunnelPort`?
Or you're thinking that inspection should be enabled on a Tunnel and they all Ports on that tunnel get auto-inspected?

Comment (IlyaBiryukov) [reply cs/src/Contracts/Tunnel.cs:130]: The tunnel options are stored as part of the tunnel and tunnel port objects. They're not a separate object.

**Review by IlyaBiryukov — APPROVED** [review]

**Pull request merged**